src/access_control.ruleset

```diff
@@ -87,13 +87,13 @@ ruleset access_control_ruleset
         }
     }
 
+    // A scan row is updated and that update must be sent to the UI.
+    //
+    // Reacts to:
+    //      updated scan record
```

Comment:
Another one with a different format. 
What is the difference?

Reply:
The format of `Reacts to: table_name.field_name` only applies to rules whose triggers are specific fields in tables. Like `scan.badge_scan` means "this rule only fires when `badge_scan` changes in a `scan` record. Meanwhile, `updated scan record` means "this rule fires when **any field** in a scan record is changed".

Reply:
```suggestion
    //      An updated row in the scan table.
```